spec/order_spec.rb

```diff
@@ -0,0 +1,29 @@
+require 'order'
+
+describe Order do
+  subject(:order) { Order.new double :menu, price_of: 9.99 }
+
+  it 'adds a dish to its items' do
+    order.add :burger
+    expected_hash = { burger: 9.99 }
```

Comment:
Setting this variable name is probably needlessly class-specific. Use `expected_items` instead --> but `expected` is a great thing to prepend to an expected value!